### ISSUE TITLE
Document getEpochTime()

### ISSUE THREAD

**Issue by donpdonp** [arduino-libraries/NTPClient#64]
going through the readme and even the .h file I was at a loss as to why there was no date functionality. Only by going through the .cpp did I find getEpochTime() - arguably the most important function in the library! Having this in the README would help newcomers.

**Comment by MrTanoshii** [comment]
Epoch time as far as NTP is concerned, is the time in seconds since January 1st 1900.
I'm also hoping they add in the date functionality, there is already a pull request for it.

**Comment by jibin2706** [comment]
@MrTanoshii  Is the offset considered in the getEpochTime()?

**Comment by MrTanoshii** [comment]
@jibin2706 
Yes, if you have a look at NTPClient.h, you will notice `#define SEVENZYYEARS 2208988800UL` which is used in the NTPClient.cpp forceUpdate(). Note that "SEVENZYYEARS" is equivalent to 70 years.

So yes, the getEpochTime() returns time from Jan 1st 1970 (Jan 1st 1900 + the 70 years offset)

**Comment by jibin2706** [comment]
I was talking about the fourth parameter from
``` NTPClient timeClient(ntpUDP, "europe.pool.ntp.org", 3600, 60000) ```
😄

**Comment by MrTanoshii** [comment]
;O `getEpochTime()` doesn't actually request make the NTP UDP request, it only returns the time since the last update. (That is you'll get an erroneous time if you didn't `update()` or` forceupdate()` before trying to `getEpochTime()`)

The fourth parameter is actually the update interval not an offset.
If you're using `update()`, a new time request is not sent out if the current `millis()` since the last update is smaller than the update interval.
If you're using `forceUpdate()`, the update interval is completely ignored

tl;dr getEpochTime is only affected by the update interval if you are using `update()`
